src/checkers/inference/InferenceQualifierHierarchy.java

```diff
@@ -325,7 +325,7 @@ public AnnotationMirror getTopAnnotation(final AnnotationMirror am) {
             return varAnnot;
         } // else
 
-        return this.getTopAnnotations().iterator().next();
+        return inferenceMain.getRealTypeFactory().getQualifierHierarchy().getTopAnnotations().iterator().next();
```

Comment:
Raise error to see if we still have the case of requiring real qual from InferQualHier.